quickstarts/microsoft.healthcareapis/workspaces/iotconnectors-with-iothub/main.bicep

```diff
@@ -0,0 +1,441 @@
+@description('Base name that is used to name provisioned resources. Should be alphanumeric, at least 3 characters and up to or less than 16 characters.')
+@minLength(3)
+@maxLength(16)
+param basename string
+
+@description('The location where the resources are deployed. This can be a different Azure region than where the resource group was deployed. For a list of Azure regions where Azure Health Data Services are available, see [Producst available by regions](https://azure.microsoft.com/explore/global-infrastructure/products-by-region/?products=health-data-services)')
+@allowed([
+  'australiaeast'
+  'canadacentral'
+  'centralindia'
+  'eastus'
+  'eastus2'
+  'koreacentral'
+  'northeurope'
+  'southcentralus'
+  'southeastasia'
+  'swedencentral'
+  'switzerlandnorth'
+  'westeurope'
+  'westus2'
+  'westus3'
+  'uksouth'
+])
+param location string
+
+@description('OPTIONAL - A principal id for a user that will be granted FHIR Data Contributor access to the FHIR service. If you do not choose to use the **fhirContributorPrincipalId** option, clear the field of any entries. To learn more about how to acquire an Azure AD user object ID, see [Find the user object ID](https://learn.microsoft.com/partner-center/find-ids-and-domain-names#find-the-user-object-id)')
+param fhirContributorPrincipalId string = ''
+
+@description('The mapping JSON that determines how incoming device data is normalized.')
+param deviceMapping object = {
```

Comment:
have you tried a portal deployment with defaultValues this large?

Reply:
@bmoore-msft 

We have and it has been successful in creating the mappings (device and destination).

![image](https://user-images.githubusercontent.com/65258541/200660450-ca13232c-d062-40eb-904a-05b2dac72a2b.png)

![image](https://user-images.githubusercontent.com/65258541/200660581-7555bec1-5f97-4bcf-ab21-4507b054c547.png)